src/layers/core/choropleth-layer/choropleth-layer.js

```diff
@@ -168,15 +168,15 @@ export default class ChoroplethLayer extends BaseLayer {
   }
 
   calculateColors(attribute) {
-    const colors = this.state.groupedVerticesColors.map(
+    const nestedColors = this.state.groupedVerticesColors.map(
       colors => colors.map(
         color => this.props.drawContour ?
           // TODO - why destructure and rebuild array?
```

Comment:
- why destruct and rebuild array?
the input color array could be vec4, instead of vec3